lib/encoder_impl.cc

```diff
@@ -286,6 +287,12 @@ void encoder_impl::set_ps(std::string ps) {
 
 		std::memset(PS, ' ', sizeof(PS));
 		std::memcpy(PS, ps.c_str(), len);
+
+        using std::cout;
+        using std::endl;
+        cout << "PS set to \"" << ps << "\"" << endl;
+        rebuild();
+        rebuild();
```

Comment:
you brought it up :) but i agree. I didn't even notice it. I should have checked and reconfigured my IDE.